st2-ask/cli-approve.txt

```diff
@@ -0,0 +1,33 @@
+vagrant@st2vagrant:~$ cat simple_approval.yaml
+---
+id: 59023cfc02ebd51154291652
+data:
+  approve: yes
+
+
+vagrant@st2vagrant:~$ st2 approval create simple_approval.yaml
++-------------+--------------------------------------------------------------+
+| Property    | Value                                                        |
++-------------+--------------------------------------------------------------+
+| id          | 59023cfc02ebd51154291652                                     |
+| data        | {                                                            |
+|             |     "approve": "yes"                                         |
+|             | }                                                            |
++-------------+--------------------------------------------------------------+
+
+vagrant@st2vagrant:~$ cat 2FA_approval.yaml
+---
+id: 59023cfc02ebd51154291653
+data:
+  second_factor: 012345
+
+
+vagrant@st2vagrant:~$ st2 approval create 2FA_approval.yaml
++-------------+--------------------------------------------------------------+
+| Property    | Value                                                        |
++-------------+--------------------------------------------------------------+
+| id          | 59023cfc02ebd51154291653                                     |
+| data        | {                                                            |
+|             |     "second_factor": "012345"                                |
+|             | }                                                            |
++-------------+--------------------------------------------------------------+
```

Comment:
Agree with this. Will update examples. And this should also allow you to provide the right values in a non-interactive way via args as well